testsuite/integration-tests/pom.xml

```diff
@@ -45,6 +45,12 @@
         <container.qualifier.manual.ssl.wrong>jbossas-manual-ssl-wrong</container.qualifier.manual.ssl.wrong>
         <container.qualifier.manual.ssl.wildcard>jbossas-manual-ssl-wildcard</container.qualifier.manual.ssl.wildcard>
         <container.qualifier.manual.ssl.sni>jbossas-manual-ssl-sni</container.qualifier.manual.ssl.sni>
+        
+        <!-- For grpc-bridge test -->
+        <version.jaxrs.example.grpc>0.0.2-SNAPSHOT</version.jaxrs.example.grpc>
+        <version.org.wildfly.core>18.1.0.Final</version.org.wildfly.core>
+        <version.weld.environment>5.0.0.CR2</version.weld.environment>
```

Comment:
We're already using 5.0.0.Final, we should remove this.

Reply:
Gone